src/data/accessibility-quiz.ts

```diff
@@ -1,4 +1,159 @@
 const accessibilityQuiz = [
+  {
+    Question: "What role does the 'alt' attribute play when adding alternative text to images on a webpage?",
+    Answer: 
+      "The 'alt' attribute describes the content of the image to users who cannot see it.",
+    Distractor1: "The 'alt' attribute enhances image aesthetics.",
+    Distractor2: "The 'alt' attribute speeds up image loading.",
+    Distractor3: "The 'alt' attribute provides translations for the image content.",
+    Explanation: 
+      "The 'alt' attribute in alternative text describes image content to users who cannot see the image.",
+    Link: "https://www.freecodecamp.org/news/common-accessibility-errors-and-how-to-fix-them/"
+  },
+  {
+    Question: "What is the purpose of the 'Chrome Accessibility Developer Tools' extension for Google Chrome?",
+    Answer: 
+    "This extension adds an Accessibility sidebar pane to the Chrome Developer tools and can be used to check accessibility properties of web elements.",
+    Distractor1: "It's a new JavaScript framework",
+    Distractor2: "It's a sorting algorithm",
+    Distractor3: "It's a type of programming paradigm",
+    Explanation: 
+    "The extension aids in inspecting and ensuring web accessibility by providing relevant information in the Developer tools.",
+    Link: "https://www.freecodecamp.org/news/accessibility-testing-tools-for-ada-compliance/"
+  },
+  {
+    Question: "Which tool or extension allows you to find accessibility defects on your web page in both Chrome and Firefox?",
```

Comment:
I would add the date since it is possible that one of these other tools might work for both chrome and firefox in the future

```suggestion
    Question: "As of 2023, which tool or extension allows you to find accessibility defects on your web page in both Chrome and Firefox?",
```